dashboard/app/models/pd/workshop.rb

```diff
@@ -183,6 +183,10 @@ def self.in_state(state)
     end
   end
 
+  def course_name
+    course == Pd::Workshop::COURSE_ADMIN ? 'Administrator' : course
+  end
+
```

Comment:
Otherwise it will show up as `Admin` in the UI, and `Administrator` in the emails.